app/src/lib/editors/win32.ts

```diff
@@ -137,6 +141,14 @@ function getRegistryKeys(
             'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\Adobe ColdFusion Builder 2016',
         },
       ]
+    case ExternalEditor.Typora:
+      return [
+        {
+          key: HKEY.HKEY_LOCAL_MACHINE,
+          subKey:
+            'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{37771A20-7167-44C0-B322-FD3E54C56156}_is1',
```

Comment:
There's also a 32-bit Windows version of Typora that installs to `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{37771A20-7167-44C0-B322-FD3E54C56156}_is1`, so I'm not sure if anyone is using it. Happy to leave that out for now, we don't have plans to support 32-bit Windows in the near-term anyway.

I also noticed both Windows installers are not signed, and reported this upstream: https://github.com/typora/typora-issues/issues/1801

Reply:
> There's also a 32-bit Windows version of Typora

I missed that! I'll add it for the sake of completion.

Tested and works on my Windows 10.

Reply:
(I have no idea how to undismiss this!)